src/GitHub.Exports/Services/Services.cs

```diff
@@ -81,15 +71,9 @@ public static IVsOutputWindowPane OutputWindowPane
             }
         }
 
-        public static DTE Dte
-        {
-            get { return GetGlobalService<DTE, DTE>(); }
-        }
+        public static DTE Dte => GetGlobalService<DTE, DTE>();
 
-        public static DTE2 Dte2
-        {
-            get { return Dte as DTE2; }
-        }
+        public static DTE2 Dte2 => Dte as DTE2;
```

Comment:
Oh, don't believe everything R# says. These are COM interfaces, the cast is ![](http://i.imgur.com/Dhlw3.gif)

It's going to be used in the future, so I keep it around, along with some other handy magical vs types (that are all probably used now, but weren't in the beginning).